tests/contrib/django/test_django_snapshots.py

```diff
@@ -42,15 +40,15 @@ def daphne_client(snapshot):
         env=env,
     )
 
-    client = Client("http://localhost:%d" % SERVER_PORT)
+    client = Client("http://localhost:%s" % port)
```

Comment:
why did this need to change from `%d` to `%s`?

Reply:
Previously on line ~35 we would convert `SERVER_PORT` from an int to a string then pass the value to a sub process. Here we were formatting it as an integer. Since this value is only used in these two places it make sense to just intialize it as a string (ie SERVER_PORT="8000" instead of SERVER_PORT=8000)